tests/cypress/support/e2e.ts

```diff
@@ -28,7 +28,7 @@ declare global {
       verifyTableRow(rowNumber: number, expectedText1?: string, expectedText2?: string|RegExp): Chainable<Element>;
       nameSpaceMenuToggle(namespaceName: string): Chainable<Element>;
       accesMenuSelection(firstAccessMenu: string, secondAccessMenu?: string, clickOption?: string): Chainable<Element>;
-      filterInSearchBox(filterText: string): Chainable<Element>;
+      filterInSearchBox(filterText: string, role?: boolean): Chainable<Element>;
```

Comment:
Is `role?: boolean` used anywhere?

Reply:
Nice Gotcha, I was working on this and it was a leftover. I will remove it in next patchset.